spec/requests/post_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  describe 'checks for response status ok' do
+    it 'returns a 200 OK status' do
+      get :index
+      expect(response).to have_http_status(:ok)
+    end
+  end
+end
+
+RSpec.describe 'Post management', type: :request do
+  it 'it directs and render users/post pages ' do
+    get '/users/2/posts'
+    expect(response).to render_template(:index)
+  end
+
+  it ' directs and renders specific user posts' do
+    get '/users/2/posts/4'
+    expect(response).to render_template(:show)
+  end
+
+  it 'renders user posts to include correct placeholder text' do
+    get '/users/2/posts'
+    expect(response.body).to include('Here is a list of post for user 2')
+  end
+
+  it 'renders page to include correct placeholder text' do
+    get '/users/2/posts/4'
+    expect(response.body).to include('Here is the post 4 for user 2')
+  end
+end
```

Comment:
- Please note that some tests are failing, kindly fix them 👍🏽 
![Screenshot from 2022-09-21 00-36-48](https://user-images.githubusercontent.com/44624138/191383763-e3fc5403-f362-40aa-9da9-8980f07c2cad.png)